src/sbm_model.jl

```diff
@@ -364,6 +407,27 @@ function initialize_sbm_model(config::Config)
             end
         end
         lateral.land.volume .= lateral.land.h .* lateral.land.width .* lateral.land.dl
```

Comment:
This line can be removed right? Is is now nested under the if statement when kinematic wave is selected (line 410&411)

Reply:
Yes, this can indeed be removed. I just noticed it when I was rebasing this branch. It is solved in commit https://github.com/Deltares/Wflow.jl/pull/182/commits/57ba9fd9b8c3a90460b7c1394176ef8d462cd019.